### DISCUSSION TITLE
Plugin for Docusaurus?

### DISCUSSION THREAD

**Discussion by rajkaramchedu** [markdoc/markdoc#264] [Q&A] [answered]
Hola - Terrific to see Markdoc out in the open. I am a tech writer, don't know much about React. I use [Docusaurus](https://docusaurus.io/docs/docs-introduction) because of ease of use, GitHub integration and Markdown. Anyone know if Markdoc can be used with Docusaurus, like with a plugin? Danke!

**Comment by mfix-stripe** [comment, marked as answer]
Hey @rajkaramchedu, there is no Markdoc plugin for Docusaurus yet, but it would be awesome if someone built one! That seems like a fantastic extension to the ecosystem.

**Reply by rajkaramchedu** [reply]
Thanks! I figured.
